main.go

```diff
@@ -148,9 +148,16 @@ func cmd() *cobra.Command {
 		Run: func(_ *cobra.Command, args []string) {
 			err := upgrade.UpgradeProvider(context, repoOrg, repoName)
 			if err != nil && context.InferVersion {
-				msg, err := createFailureIssue(context, repoOrg, repoName)
-				if err != nil {
-					fmt.Println(msg)
+				// $GITHUB_ACTION is a default env var within github
+				// actions, but is unlikely to be defined elsewhere.
+				//
+				// We do this to test that we are being run inside a GH
+				// action.
+				if _, ci := os.LookupEnv("GITHUB_ACTION"); ci {
```

Comment:
We only want to run this in Github actions CI. Otherwise the link we set doesn't make sense.